.github/workflows/flaky-tests-stalebot-grooming.yml

```diff
@@ -0,0 +1,23 @@
+name: Flaky test auto-grooming
+# Automatically closes flaky test issues when they are over a week old with no activity.
+on:
+    schedule:
+        - cron: '0 0 * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Close Stale Issues
+              uses: actions/stale@v9.0.0
+              with:
+                  only-labels: 'metric: flaky e2e test, team: Vortex'
+                  days-before-stale: 5
+                  days-before-close: 7
+                  stale-issue-label: 'metric: stale flaky e2e test report'
```

Comment:
Applies a stale label (this is mostly to warn folks that they may not want to spend a bunch of time investigating because this may not be a legit issue).